### ISSUE TITLE
LogWarnings from PFClusterEMEnergyCorrector: Flag 5 is not recognized

### ISSUE THREAD

**Issue by slava77** [cms-sw/cmssw#19801]
this warning is showing up after #19307 was integrated

e.g. in CMSSW_9_3_0_pre2 in workflow 136.780 in step3

```
%MSG-w PFClusterEMEnergyCorrector:   CorrectedECALPFClusterProducer:particleFlowClusterECAL  18-Jul-2017 18:53:11 CEST Run: 297227 Event: 18725
We can only correct regions readout in ZS (flag 1) or FULL readout (flag 3). Flag 5 is not recognized.
Assuming FULL readout and continuing
```

the frequency varies and probably depends on the occupancies per event:
- the 136.78 has about 20% of events with at least one warning
- 299042 has it a few times per event

@rafaellopesdesa @jainshilpi @emanueledimarco @argiro 
if flag=5 is a normal flag and it is known how to treat it, please adjust the logic in the code.
If the warning is not essential, please demote it to logInfo (at least for this frequent flag=5 case).

**Comment by slava77** [comment]
assign reconstruction

**Comment by jainshilpi** [comment]
@slava77 I am looking into it. Will update.

**Comment by slava77** [comment]
fixed in #19827

I will +1 and close once the PR is merged

**Comment by slava77** [comment]
+1

#19827 is now merged in 93X.
